extensions/cornerstone/src/utils/measurementServiceMappings/EllipticalRoi.js

```diff
@@ -58,6 +71,7 @@ const EllipticalRoi = {
       FrameOfReferenceUID,
       referenceSeriesUID: SeriesInstanceUID,
       referenceStudyUID: StudyInstanceUID,
+      displaySetInstanceUID: displaySet.displaySetInstanceUID,
```

Comment:
Right. Because MeasurementService now tracks this for the sanity of all involved.
While it's serializable, it'd be nonsense to persist this anywhere and try to re-load it.

Reply:
It is not. Its for session use only. New displaySetInstanceUIDs are created on refresh anyway.